a5/Dockerfile

```diff
@@ -1,6 +1,7 @@
 # rosindustrial/a5:kinetic
-FROM rosindustrial/noether-webtools:kinetic
+FROM rosindustrial/noether:kinetic
 MAINTAINER Austin.Deric@gmail.com
 RUN apt-get update && apt-get install -y \
-  ros-kinetic-stage \
-  sudo
+  ros-kinetic-stage
+RUN apt-get clean && \
+  rm -rf /var/lib/apt/lists/*
```

Comment:
My apologies for the many comments, but I think you're missing the `&&` after `ros-kinetic-stage` here. As it is, `apt-get` will get an additional 2 packages: `apt-get` and `clean`, which don't exist.